[2024-09-25T09:59:52.534+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T09:59:52.674+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.orders scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T09:59:52.763+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.orders scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T09:59:52.764+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-25T09:59:52.877+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): warehouse.step_1.extract_transform.orders> on 2024-09-24 00:00:00+00:00
[2024-09-25T09:59:52.912+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=385) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T09:59:52.923+0000] {standard_task_runner.py:72} INFO - Started process 401 to run task
[2024-09-25T09:59:52.923+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'warehouse.step_1.extract_transform.orders', 'scheduled__2024-09-24T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpzvulx_1g']
[2024-09-25T09:59:52.934+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask warehouse.step_1.extract_transform.orders
[2024-09-25T09:59:53.189+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.warehouse.step_1.extract_transform.orders scheduled__2024-09-24T00:00:00+00:00 [running]> on host 639d778bfb8f
[2024-09-25T09:59:53.637+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='warehouse.step_1.extract_transform.orders' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:00:00+00:00'
[2024-09-25T09:59:53.641+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T09:59:53.687+0000] {base.py:84} INFO - Retrieving connection 'staging_db'
[2024-09-25T09:59:54.140+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T10:00:04.800+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T10:00:04.866+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T10:00:04.885+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T10:00:04.886+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=warehouse.step_1.extract_transform.orders, run_id=scheduled__2024-09-24T00:00:00+00:00, execution_date=20240924T000000, start_date=20240925T095952, end_date=20240925T100004
[2024-09-25T10:00:04.936+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T10:00:05.010+0000] {taskinstance.py:3900} INFO - 7 downstream tasks scheduled from follow-on schedule check
[2024-09-25T10:00:05.014+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
